src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualHub.md

```diff
@@ -0,0 +1,140 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/get-azurermvirtualhub
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualHub
+
+## SYNOPSIS
+Gets an Azure VirtualHub by Name and ResourceGroupName or lists all Virtual Hubs by ResourceGroupName/Subscription.
+
+## SYNTAX
+
+```
+Get-AzureRmVirtualHub [-Name <String>] [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```

Comment:
Make sure all synopsis and description fields are filled out.

Reply:
Fixed.